pkg/daemon/daemon.go

```diff
@@ -1400,6 +1400,10 @@ func (dn *Daemon) checkStateOnFirstRun() error {
 		return fmt.Errorf("error detecting previous SSH accesses: %w", err)
 	}
 
+	if err := dn.removeRollback(); err != nil {
+		return fmt.Errorf("Failed to remove rollback: %w", err)
```

Comment:
should this be informational instead of error?

Reply:
Previous version of this code had it as a error too. But it's an easy change. I would imagine that if we get an error from rpm-ostree we should fail. It could mean something else is using the files? or something is wrong with the file system?